.github/workflows/gcloud-gke-test.yml

```diff
@@ -0,0 +1,75 @@
+name: test acorn on GKE
+on:
+  push:
+    paths-ignore:
+      - docs/**
+    branches:
+      gh-actions-gke
+  schedule:
+    - cron: '00 7 * * *'   # time in UTC
+
+jobs:
+  acorn-test-gke:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read 
+
+    steps:
+      - name: install curl
+        run: |
+          sudo apt update
+          sudo apt install -y curl 
+      - name: install helm
+        run: |
+          mkdir -p /tmp/helm
+          curl --silent --location https://get.helm.sh/helm-v3.10.0-linux-amd64.tar.gz | tar xz -C /tmp/helm linux-amd64/helm
+          sudo mv /tmp/helm/linux-amd64/helm /usr/local/bin/helm
+
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.19
+      - run: make setup-ci-env
+      - run: make validate-ci
+      - run: make validate
+      - run: make build
+      - run: sudo install -o root -g root -m 0755 ./bin/acorn /usr/local/bin/acorn
+
+      - name: 'Authenticate to Google Cloud'
+        uses: 'google-github-actions/auth@v0'
+        with:
+          credentials_json: '${{ secrets.GCLOUD_CREDENTIALS }}'
+
+      - name: 'Set up Cloud SDK'
+        uses: 'google-github-actions/setup-gcloud@v0'
+           
+      - name: 'Create GKE cluster'
+        run: |
+          gcloud container clusters create --zone "${REGION}" --cluster-version "${GKE_K8S_VERSION}"  "${GKE_CLUSTER_NAME}"  --num-nodes=1  --issue-client-certificate  --enable-legacy-authorization
```

Comment:
Where are these ENV var values coming from?

Reply:
we need to create a service account with the roles as in PR description, and then we create service account key, set GH secret with the json file created as a result.

for the region, we can any GCP region of our choosing.